File: Beginner/Java/Math/FizzBuzz.java
class Solution {
    public List<String> fizzBuzz(int n) {
        List<String> fb = new ArrayList<String>(n);
        String f = "Fizz", b = "Buzz";
        for (int i = 1; i <= n; i ++) {
            if (i % 15 == 0)
                fb.add(f + b);
            else if (i % 3 == 0)
                fb.add(f);
            else if (i % 5 == 0)
                fb.add(b);
            else
                fb.add(String.valueOf(i));
        }
        return fb;
    }
}

File: Beginner/Python/Math/FizzBuzz.py
class Solution(object):
    def fizzBuzz(self, n):
        """
            :type n: int
            :rtype: List[str]
            """
        L = []
        if n > 0:
            for i in range(1, n + 1):
                string = ''
                if i % 3 == 0:
                    string += 'Fizz'
                if i % 5 == 0:
                    string += 'Buzz'
                if i % 3 != 0 and i % 5 != 0:
                    string += str(i)
                L.append(string)
        return L
